### DISCUSSION TITLE
Can I ask a question?

### DISCUSSION THREAD

**Discussion by WOODUCK** [dannagle/PacketSender#262] [Q&A] [answered]
Hi, this is Woo Seok who wants to be a master student in South Korea.

I am researching EGSE (Electrical Ground Support Equipment) using Ball aerospace COSMOS 4.5.0 and Packet Sender to test internal tests.

I succeed INT Data and FLOAT Data.

However, the next step is filling all parameters communicating between  GUI in COSMOS and Packet Sender. I tried to open Packet Sender again.

But I got an error message Subject is 'Port bind error' and 
the content was 'Packet Sender attempted (and failed) to bind to port 0.
- Are you running multiple instances?
- Trying to bind to a missing custom IP'
My Packet Sender was set Address (127.0.0.1), Write port is 8877 and Read port is 8888,

After I got this message, I can`t parse data.

So if you guys get this error message, can I get a solution??

Thanks for reading my question.

I attached my pics to understand easier.

![pic 1 error message](https://user-images.githubusercontent.com/86736519/144388995-950e6a3b-f91b-4b42-bfb8-d1739d2abf01.jpg)
![pic 2 setting](https://user-images.githubusercontent.com/86736519/144389002-bfa0a4ed-bb2e-42b1-b115-2dcf4aeae85e.jpg)
![pic 3 problem](https://user-images.githubusercontent.com/86736519/144389005-1b8b579b-7474-41c5-85e8-3d1fc25825ee.jpg)

**Comment by dannagle** [comment, marked as answer]
Packet Sender's client interface should continue to work OK despite this error. 

Some ideas...
Could you possibly have 2 Packet Senders running trying to use the same port?

This error typically occurs because you are trying to bind to a specific port (Go to File --> Settings, and look under TCP and/or SSL servers), and other app (does not have to be Packet Sender) has taken it.

This also sometimes happens if you are trying to bind to a low port number (less than 1024). This requires admin permissions. 


If you are having trouble with multiple instances of Packet Sender, you can try using the installed version along with the portable version.

**Reply by WOODUCK** [reply]
okay, I try to apply your advice. Thanks for letting me know.
